encoding/json/decode.go

```diff
@@ -565,6 +567,24 @@ func (d *Decoder) decodeWord64(valueJSON any) cadence.Word64 {
 	return cadence.NewMeteredWord64(d.gauge, i)
 }
 
+func (d *Decoder) decodeWord128(valueJSON any) cadence.Word128 {
+	value, err := cadence.NewMeteredWord128FromBig(
+		d.gauge,
+		func() *big.Int {
+			bigInt := d.decodeBigInt(valueJSON)
+			if bigInt == nil {
+				// TODO: propagate toString error from decodeBigInt
+				panic(errors.NewDefaultUserError("invalid Word128: %s", valueJSON))
```

Comment:
Is this a TODO for this PR? Also, please add a test case for this line, and the other uncovered lines below

Reply:
> Is this a TODO for this PR? 

No, copy-paste from the implementation of `decodeUInt128` & `decodeInt128`.

Ack on the test case part, I'll add them soon.

Reply:
@turbolent  Many of these uncovered lines are also not covered for other types. For e.g. `convert` function in `runtime/interpreter/interpreter.go` is uncovered. I think it would be nice to tackle that in a separate issue. Filed https://github.com/onflow/cadence/issues/2506 for the same, will be happy to tackle it post the grant milestones.